dashboard/app/helpers/course_block_helper.rb

```diff
@@ -178,6 +178,11 @@ def self.get_tall_course_block(id)
       Script::TWENTY_HOUR_NAME => {
         title: I18n.t('upsell.20hour.title'),
         body: I18n.t('upsell.20hour.body')
+      },
+      Script::OCEANS_NAME => {
+        url: CDO.code_org_url('/oceans'),
+        title: 'AI for Oceans',
+        body: 'Learn about machine learning and ethical use of AI. #CSforGood'
```

Comment:
I'm not super concerned about loc'ing this since it's English only, but where should those strings live? There are a few different options already in this file and I didn't know which path to match.

Reply:
Would these work?
```title: data_t_suffix('script.name', id, 'title'),
body: data_t_suffix('script.name', id, 'description_short')
```
We have a work item to get the script description in very soon.